cost-analyzer/values.yaml

```diff
@@ -165,6 +165,15 @@ prometheus:
             - expr: sum(container_memory_working_set_bytes{container_name!="POD",container_name!=""})
               record: kubecost_cluster_memory_working_set_bytes
 
+networkCosts:
+  enabled: true
+  image: gcr.io/guestbook-227502/kubecost-network-costs:v26
+  resources:
+    #requests:
```

Comment:
Pretty big CPU request for a daemonset. Any chance we could dogfood our own app to come up with something good here?

Reply:
Excellent